File: app/src/main/java/com/example/testlunu/net/acu/cityinfo/Region.java
package com.example.testlunu.net.acu.cityinfo;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Region {

    @SerializedName("ID")
    @Expose
    public String iD;
    @SerializedName("LocalizedName")
    @Expose
    public String localizedName;
    @SerializedName("EnglishName")
    @Expose
    public String englishName;

}

File: app/src/main/java/com/example/testlunu/net/acu/cityinfo/Region.kt
package com.example.testlunu.net.acu.cityinfo

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class Region (
    @SerializedName("ID")
    @Expose
    var iD: String?,
    @SerializedName("LocalizedName")
    @Expose
    var localizedName: String?,
    @SerializedName("EnglishName")
    @Expose
    var englishName: String?
    )